src/main/java/wolox/training/models/Book.java

```diff
@@ -10,10 +11,15 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.ManyToMany;
-import wolox.training.utils.ValidationPreconditionUtil;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
 
 @Entity
 @ApiModel(description = "Books from the OpenLibrary")
+@RequiredArgsConstructor
+@Getter
```

Comment:
It would be better to have `@Data` annotation instead of this 3 annotations, because `@Data` adds  `@RequiredArgsConstructor ￼@Getter @Setter` but also adds `@ToString @EqualsAndHashCode`!